index.html

```diff
@@ -4,9 +4,226 @@
+            <div class="show-all-btn mt-4 d-flex justify-content-center">
+                <a href="#" class="white-color">see the whole dishes</a>
+            </div>
+        </div>
+    </section>
+    <section class="features py-5 mb-5">
+        <div class="container">
+            <div class="title text-center d-flex justify-content-center">
+                <h2>Features Cooks</h2>
+            </div>
+            <div id="features-cook" class="row">
+               
+            </div>
+            <div class="mt-4 d-flex justify-content-center">
+                <a class="read-btn" >
+                    <span class="more">
+                            More
+                        <span>
+                            <i class="fa-solid fa-arrow-down"></i>
+                        </span>
+                    </span>
+                    <span class="less d-none">
+                        Less
+                        <span>
+                            <i class="fa-solid fa-arrow-up"></i>
+                        </span>
+                    </span>
+                </a>
+            </div>
+        </div>
+    </section>
+    
+    <section class="partners py-5 black-background">
+        <div class="container">
+            <div class="title text-center d-flex justify-content-center">
+                <h2 class="gray-color">partners</h2>
+            </div>
+            <div class=" icon d-flex  flex-wrap justify-content-center align-items-baseline mt-5 gray-color">
+                
+                    <div>
+                        
+                        <img src="images/Williams-Sonoma-EDI.png"  alt="Williams Sonoma">
+                    </div>
+                    <div>
+                        <img src="images/LH-Logos_LH-All+-+Color+copy.png" alt="local haravest" >
+                    </div>
+                    
+                    <div>
+                        <img src="images/Tripadvisor-Emblem.png" alt="Trip advisor" >
+                    </div>
+                   <div>
+                    <img class="none-filter" src="images/foodandwinelogowhite-ac7f9b70c63a3dcb0160ed30ab19adcde87c48e709183673f2518af87f3a171f.png" alt="Food & Wine">
+                   </div>
+            
+                   
+            </div>
+        </div>
+    </section>
+    <footer class="p-5">
+        <div class="container">
+            <div class="row ">
+                <div class="col-3 image">
+                    <img src="images/logo-chef.png" alt="Hire Chef" />
+                </div>
+                <div class="col-9  copywrite">
+                    <p>Hire Chef. Some Rights Reserved.</p>
+                    <p class="p-0 m-0 description">The design used in this project is based on the original work by [Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and has been modified to fit the requirements of the project.</p>
+                </div>
+            </div>
+        </div>
+    </footer>
```

Comment:
As asked by the previous code reviewer please consider aligning the content inside the footer in the center to meet the design guidelines currently, it's okay in the desktop version but does not fit well in the mobile version.

## YOUR DESIGN:--


![image](https://user-images.githubusercontent.com/94223367/233469102-f43c8be8-924b-4eeb-ac49-234003a1aa2e.png)
![image](https://user-images.githubusercontent.com/94223367/233469156-0c0ad28c-44cb-4d67-aa71-e48e06a387f5.png)